GliaWidgets/Component/Bubble/BubbleStyle.swift

```diff
@@ -6,16 +6,21 @@ public class BubbleStyle {
     /// Style of a badge shown on the bubble.
     public var badge: BadgeStyle?
 
+    /// Style of a visitor's on hold state overlay.
+    public var onHoldOverlay: OnHoldOverlayStyle?
```

Comment:
Perhaps I miss something, however I'd like to clarify why `onHoldOverlay` is optional and is variable?
For me, it looks like if this style is nil then `bubble` view will not show on hold state at all - [code](https://github.com/salemove/ios-sdk-widgets/pull/248/files#diff-811f3bc73367f56c237557e63c8e38abab3438cf50662d4dfea7ac21da63c7b7R59).
If we would like to apply default value, I think it should be some default `OnHoldOverlayStyle` and it should not lead to incorrect appearance of Bubble view.
Does it make sense?

Reply:
I think the same reason why badge style is optional. It was developed after and we didn't want to force integrators to implement them with errors. But default value is very good idea.